.github/CODEOWNERS

```diff
@@ -42,7 +42,5 @@
 /Xamarin.Forms.Core/ @StephaneDelcroix
 
 # Blazor Desktop
-/src/BlazorWebView/                                     @Eilon
-/src/Templates/src/templates/maui-blazor/               @Eilon
-
-
+/src/BlazorWebView/                                     @Eilon @dotnet/dotnet-maui-blazor-eng
```

Comment:
Actually can you remove me from the explicit list? I'm already in the group anyway.